src/webcomponents/commons/filters/hpo-accessions-filter.js

```diff
@@ -38,16 +35,17 @@ export default class HpoAccessionsFilter extends LitElement {
         return {
             "annot-hpo": {
                 type: Object
+            },
+            "cellbaseClient": {
+                type: Object
             }
         };
     }
 
     _init() {
```

Comment:
As in `go-accessions-filter.js` you made the `_init` method private, I would suggest doing the same there.